cnvlib/export.py

```diff
@@ -268,12 +268,12 @@ def segments2vcf(segments, ploidy, is_reference_male, is_sample_female):
     # TODO be more clever about this
     for (_idx, out_row), (_idx, abs_row) in zip(out_dframe.iterrows(),
                                                 abs_dframe.iterrows()):
-        if out_row["ncopies"] == abs_row["expect"]:
+        if out_row["ncopies"] == abs_row["expect"] or out_row["probes"] == "_":
```

Comment:
Sounds reasonable to me. cnvkit output the cns file with the underscore, so I thought that's what you used but happy to have this more general. Do you want me to add that fix?